features/step_definitions/inbox_steps.rb

```diff
@@ -0,0 +1,18 @@
+Given(/^I am on the welcome page$/) do
+  visit welcome_index_path
+end
+
+Then(/^I should be on inbox page$/) do
+  visit mailbox_inbox_path
+end
```

Comment:
No. "I should be" requires an expectation. This step does exactly what the step on line 1 does - we should never have two step definitions with exactly the same actions.